parm/parm_fv3diag/diag_table_aer

```diff
@@ -0,0 +1,366 @@
+#20161003.00Z.C96.64bit.non-mono
```

Comment:
If this file is added, why is the file: 
diag_table_cpl (https://github.com/NOAA-EMC/global-workflow/blob/feature/coupled-crow/parm/parm_fv3diag/diag_table_cpl ) not added?  I don't think any of these are needed/required for ATM-only but I don't know why only some of the coupled and not all the coupled updates are getting added? 

Also there is a change for the coupled side in : https://github.com/NOAA-EMC/global-workflow/blob/feature/coupled-crow/parm/parm_fv3diag/data_table but again not needed for ATM-only.

Reply:
OK now added that file. I'm aware of the changes in data_table but decided to not include it because I don't know if it will affect atm-only run.